ckanext/cfpb_extrafields/plugin.py

```diff
@@ -88,3 +291,10 @@ def update_config(self, config):
         # Add this plugin's templates dir to CKAN's extra_template_paths, so
         # that CKAN will use this plugin's custom templates.
         tk.add_template_directory(config, 'templates')
+
+        # Register this plugin's fanstatic directory with CKAN.
+        # Here, 'fanstatic' is the path to the fanstatic directory
+        # (relative to this plugin.py file), and 'example_theme' is the name
+        # that we'll use to refer to this fanstatic directory from CKAN
+        # templates.
+        #tk.add_resource('fanstatic', 'example_theme')
```

Comment:
Is this here for future needs?  It doesn't look like we're using it anywhere else.